### ISSUE TITLE
Dynamo_Toolkit: Support for Dynamo 2.3+

### ISSUE THREAD

**Issue by kayleighhoude** [BHoM/Dynamo_Toolkit#275]
#### Description:
Issue occurring in Dynamo between sessions where the entire canvas appears as wires without nodes (this is an Autodesk issue not a BHoM one):
![image](https://user-images.githubusercontent.com/34316256/98593888-d85d3c00-22a1-11eb-9072-c6a1c18da094.png)

The proposed solution by Autodesk is to upgrade your version of Dynamo. Considering that I'm currently in 2.3, the highest supported BHoM version, I would not be able to access BHoM if I upgraded to 2.8 (latest Dynamo). Would we be able to add support for these later versions of Dynamo?

**Comment by adecler** [comment]
Yes, that shouldn't be too hard (I am potentially going to regret saying this) as long as it is just a question of adding an extra output folder for 2.8. 

Will do that as soon as https://github.com/BHoM/Dynamo_Toolkit/pull/274 is merged.

**Comment by pawelbaran** [comment]
I have just given it a shot with copy-pasting the assemblies to Dynamo 2.6, unfortunately with no success. @adecler, could we please prioritize this issue early 4.1, to make sure we have support for Revit 2021 as the projects start rolling out? Thanks!

**Comment by JosefTaylor** [comment]
Hi @adecler, is this still an issue? I'm trolling the archives to see if there's an issue with Revit 2022 and wondered if this is related.

**Comment by FraserGreenroyd** [comment]
@JosefTaylor we're not currently supporting dynamo past 2.3 or dynamo with Revit 2022. There's a discussion on the BHoM discussion page with some more information that may help.
